### PR TITLE
setup project two

### PR DESCRIPTION
**Pull request by RSC01**
Check This Second SetUp

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
+    {/* resolve testing comment for index  js */}
```
Comment (RSC01) [inline src/index.js:9]: What is this ?

**Comment by GhusarParth** [comment]
ok resolve done

File: src/index.js (hunk 1 above)
Comment (GhusarParth) [inline src/index.js:9]: remove comments

Comment (RSC01) [reply src/index.js:9]: ok

**Pull request closed without merging**